docs/token-contracts/vesting.md

```diff
@@ -0,0 +1,346 @@
+---
+id: vesting
+title: Getting started with the Vesting contract
+sidebar_label: Vesting Contract Quick Start
+---
+
+# Introduction
```

Comment:
AFAIU in the articles, we start at level `##` (and the `h1` is left for the global title).